java.lang.IllegalArgumentException: Cannot join less than 2 cells error when adding footer to grid

I'm using GridHelpers 1.1.1. and Vaadin 24.1.4.

I add a footer to a grid with just one column. The error (java.lang.IllegalArgumentException: Cannot join less than 2 cells) comes from FooterToolbarGridHelper.java:28 (from stacktrace) .

Stack trace: 

```
Caused by: java.lang.IllegalArgumentException: Cannot join less than 2 cells
at com.vaadin.flow.component.grid.AbstractRow.join(AbstractRow.java:276)
at com.vaadin.flow.component.grid.FooterRow.join(FooterRow.java:86)
at com.vaadin.flow.component.grid.FooterRow.join(FooterRow.java:30)
at com.vaadin.flow.component.grid.AbstractRow.join(AbstractRow.java:215)
at com.vaadin.flow.component.grid.FooterRow.join(FooterRow.java:30)
at com.flowingcode.vaadin.addons.gridhelpers.FooterToolbarGridHelper.setFooterToolbar(FooterToolbarGridHelper.java:28)
at com.flowingcode.vaadin.addons.gridhelpers.GridHelper.addToolbarFooter(GridHelper.java:346)
at com.flowingcode.vaadin.addons.gridhelpers.AddToolbarFooterDemo.<init>(AddToolbarFooterDemo.java:53)
... 72 more
```

--- a/.gitignore
+++ b/.gitignore
@@ -19,3 +19,5 @@ types.d.ts
 /pnpm-lock.yaml
 /pnpmfile.js
 /src/main/dev-bundle
+vite.generated.ts
+vite.config.ts

--- a/src/main/java/com/flowingcode/vaadin/addons/gridhelpers/FooterToolbarGridHelper.java
+++ b/src/main/java/com/flowingcode/vaadin/addons/gridhelpers/FooterToolbarGridHelper.java
@@ -25,7 +25,11 @@ class FooterToolbarGridHelper implements Serializable {
     }
     if (footerCell == null) {
       FooterRow fr = grid.appendFooterRow();
-      footerCell = fr.join(grid.getColumns().toArray(new Grid.Column[0]));
+      if(grid.getColumns().size() > 1) {
+        footerCell = fr.join(grid.getColumns().toArray(new Grid.Column[0]));
+      } else {
+        footerCell = fr.getCell(grid.getColumns().get(0));
+      }
     }
     toolBar.getElement().setAttribute("fcGh-footer", true);
     footerCell.setComponent(toolBar);

--- a/src/test/java/com/flowingcode/vaadin/addons/gridhelpers/test/FooterToolbarTest.java
+++ b/src/test/java/com/flowingcode/vaadin/addons/gridhelpers/test/FooterToolbarTest.java
@@ -1,0 +1,30 @@
+package com.flowingcode.vaadin.addons.gridhelpers.test;
+
+import com.flowingcode.vaadin.addons.gridhelpers.GridHelper;
+import com.vaadin.flow.component.grid.Grid;
+import com.vaadin.flow.component.orderedlayout.HorizontalLayout;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.Test.None;
+
+public class FooterToolbarTest {
+  
+  private Grid<Bean> grid;
+
+  private HorizontalLayout toolbarFooter;
+  
+  private class Bean {}
+  
+  @Before
+  public void before() {
+    grid = new Grid<>(Bean.class, false);
+    grid.addColumn(x -> x).setHeader("Header");
+    toolbarFooter = new HorizontalLayout();    
+  }
+  
+  @Test(expected = None.class)
+  public void addToolbarFooter_toolbarFooterIsShown() {
+    GridHelper.addToolbarFooter(grid, toolbarFooter);
+  }
+
+}

